doc/source/language/variables.rst

```diff
@@ -113,3 +165,171 @@ Sets a variable to ``FALSE``. This is useful for the ``RCS`` and ``SAS`` binding
 
     RCS OFF.  // Turns off the RCS
 
+This follows the same rules as :ref:`SET <set>`, in that if the variable in
+question doesn't already exist, it will end up creating it as a global 
+variable.
+
+.. _scope:
+
+Scoping rules
+-------------
+
+.. note::
+    .. versionadded:: 0.17
+        In prior versions of kerboscript, all identifiers other than
```

Comment:
I broke FOR loops when I changed the scoping rules, it turns out, but that's okay because I just changed the FOR's secret iterator to now be a local scope variable anyway, as it should be, and that fixed it.